account_invoice_validation_queued/migrations/14.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not openupgrade.table_exists(env.cr, "account_move_queue_job_rel"):
+        return
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO account_move_validation_job_rel
```

Comment:
Why is this needed? Can't you preserve the old name or rename the table in pre?

Reply:
@pedrobaeza means like that?

Reply:
Not really. Now I see both post and pre migration scripts doing the insert. My question about why is this needed still unanswered, and without it, I can't recommend you the actions to take.

Reply:
Right. That's something from 13.0 and I believe it can be removed. I've pushed without the migration folder.
Thank you @pedrobaeza